examples/docs-wisdom-8/index.txt

```diff
@@ -0,0 +1,13 @@
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+Checked 1 theorems; No failures!
+warning[RW0004]: There are no publications in the application.
```

Comment:
This file is inaccurate now. You need to regenerate it with `scripts/gen-test-output.sh`